jsonapi_requests/auth.py

```diff
@@ -0,0 +1,7 @@
+import flask
+
+
+class FlaskForwardAuth:
```

Comment:
How about subclassing `AuthBase` from `requests`?
http://docs.python-requests.org/en/master/user/authentication/#new-forms-of-authentication